example/www/js/index.js

```diff
@@ -46,21 +68,33 @@ var app = {
 		});
 		*/
 		cordova.plugins.radar.requestPermissions(true);
-		/*
-		cordova.plugins.radar.getPermissionsStatus((status) => {
-			alert(JSON.stringify(status));
+		cordova.plugins.radar.getPermissionsStatus((result) => {
+		  this.log("getPermissionsStatus: " + JSON.stringify(result));
 		});
-		cordova.plugins.radar.trackOnce((result) => {
-		  alert(JSON.stringify(result));
+		cordova.plugins.radar.trackOnce({ desiredAccuracy: "medium", beacons: true }, (result) => {
+		  this.log("trackOnce: " + JSON.stringify(result));
+		});
+		cordova.plugins.radar.isTracking((result) => {
+			this.log("isTracking: " + JSON.stringify(result));
+		})
+		cordova.plugins.radar.startTrackingContinuous();
+		cordova.plugins.radar.getTrackingOptions((result) => {
+			this.log("getTrackingOptions: " + JSON.stringify(result));
 		});
 		cordova.plugins.radar.searchPlaces({
-			radius: 10000,
-			chains: ['starbucks'],
-			limit: 10,
+      near: {
```

Comment:
weird indentation?

Reply:
weird, on my vscode, it looks nice, i will check again.